src/main/java/com/bpodgursky/jbool_expressions/Variable.java

```diff
@@ -54,4 +57,15 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(value);
   }
+  public Set<K> getAllK(){
+	  Set<K> set=new HashSet<K>();
+	  set.add(value);
```

Comment:
return Collections.singleton(value) would be (very slightly) more efficient here.